shopitems/gloomitems.py

```diff
@@ -0,0 +1,57 @@
+from ... import (
+  shopsdk.digitalcontentsender
+  shopsdk.install
+  shopsdk.orderphotos
+  shopsdk.orderprintful
+  shopsdk.setup
+  shopsdk.utilsnotbyme
+)
+class GloomDLCShopItems(l, o, p, r, s, t):
+  def GloomDownloadableItem0(l, o, p, r, s, t):
```

Comment:
Please do not obfuscate variable names.

Reply:
Alright, I can probably change em back :)